test/specs/components/Component/Component-test.ts

```diff
@@ -0,0 +1,33 @@
+import { mountWithProvider as mount } from 'test/utils'
+
+import Component from 'src/components/Component/Component'
+import { isConformant } from 'test/specs/commonTests'
+
+describe('Component', () => {
+  const createComponentComp = (factoryFn: Function, val, options?) =>
+    mount(factoryFn(val, options)).find(Component)
+
+  xdescribe('is conformant', () => {
```

Comment:
Should this be `xdescribe`?

Reply:
yes, we still cannot reenable the tests due to #502 as info files are not created for Slot so conformance tests fail